snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/charts.helm.k8s.io_snykmonitors_crd.yaml

```diff
@@ -1,24 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
if the intention is to rename the file can we use `git mv` to preserve the git history?